common/metrics/src/public_metrics.rs

```diff
@@ -3,6 +3,8 @@
 
 use lazy_static::lazy_static;
 
+// A list of metrics which will be made public to all the partners
 lazy_static! {
-    pub static ref PUBLIC_METRICS: Vec<String> = vec!["libra_network_peers".to_string()];
+    pub static ref PUBLIC_METRICS: Vec<String> =
```

Comment:
Not really related to this PR, but is there a reason why we need this to be a vec? might actually be better if this was a const array with &'static str entries. especially since this shouldn't be able to be changed during runtime.